web/src/containers/UserProfile.js

```diff
@@ -0,0 +1,208 @@
+import PropTypes from "prop-types";
+import React, { useEffect } from "react";
+import UserService from "../services/user.service";
+import {
+  TextField,
+  Select,
+  Box,
+  Grid,
+  FormControl,
+  InputLabel,
+  Button,
+  Typography,
+} from "@material-ui/core";
+import { CloudUpload } from "@material-ui/icons";
+import { useFormik } from "formik";
+import { countries } from "../util/countries";
+import KycButton from "../components/KycButton";
+function UserProfile() {
+  useEffect(async () => {
```

Comment:
Is this working? useEffect can't handle async function as argument i guess (cf. other useEffects)